pkg/config/config.go

```diff
@@ -38,6 +38,8 @@ type Runtime struct {
 	// PodAnnotations is a list of pod annotations passed to both pod sandbox as well as
 	// container OCI annotations.
 	PodAnnotations []string `toml:"pod_annotations" json:"PodAnnotations"`
+	// ContainerAnnotations is a list of container annotations passed through to the OCI config of the containers.
```

Comment:
Add the following:
```
// Container annotations in CRI are usually generated by other Kubernetes node components (i.e.,
// not users). Currently, only device plugins populate the annotations.
```

Reply:
done

Reply:
Sorry, I mean adding this after or in front of you original comment. And please also update `docs/config.md` with the same text.